provider/resource_keycloak_openid_client.go

```diff
@@ -180,6 +181,11 @@ func resourceKeycloakOpenidClient() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"permissions_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
```

Comment:
Would Having resourceKeycloakOpenidClientPermissions pointing a client not make this obsolete?
I would let the resourceKeycloakOpenidClientPermissions handle the enable of disable of the permissions on the client.
Even in keycloak I think the permissions are located in the wrong location. It is really a resource in it own right
What do you think?

Reply:
I added `permissions_enabled` in` resourceKeycloakOpenidClient` to be able to enable / disable permissions independent of the permissions configuration themselves.

We can indeed consider that the presence of `resourceKeycloakOpenidClientPermissions` activates the permissions on the client.
To be able to manage its deactivation, I will keep the `enabled` parameter in` resourceKeycloakOpenidClientPermissions` which with the value `false` will delete the resource